### PR TITLE
`npx ember-cli-update`

### PR DESCRIPTION
**Pull request by dcyriller**
Also: 
- upgrade ember-template-lint to v2-beta
- cherry-pick ember-cli upgrade from https://github.com/ember-template-lint/ember-cli-template-lint/pull/553

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 root = true
 
+
```
Comment (rwjblue) [inline .editorconfig]: Seems vaguely odd, I didn't realize we had two newlines back to back in this file...

File: LICENSE.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Robert Jackson
+Copyright (c) 2019
```
Comment (rwjblue) [inline LICENSE.md]:
```suggestion
Copyright (c) 2016-2019 Robert Jackson
```

File: package.json [hunk 3]
```diff
@@ -35,39 +36,44 @@
     "chalk": "^3.0.0",
     "debug": "^4.0.1",
     "ember-cli-version-checker": "^3.0.1",
-    "ember-template-lint": "^1.2.0",
+    "ember-template-lint": "github:ember-template-lint/ember-template-lint#next",
```
Comment (rwjblue) [inline package.json]: Can you tweak to the released beta?

File: .editorconfig (hunk 1 above)
Comment (dcyriller) [reply .editorconfig]: https://github.com/ember-cli/ember-cli/pull/9013 😉

File: package.json (hunk 3 above)
Comment (dcyriller) [reply package.json]: It was failing on beta.4, I've released a beta.5 for template-lint

**Pull request merged**